### PR TITLE
Use issues instead of PRs

### PR DESCRIPTION
**Pull request by bbarker**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,4 +12,4 @@ Check out these milestones for the list of RFCs that are planned to be or are cu
 1. Fork and clone this repository.
```
Comment (Yardanico) [inline README.md:12]: This is not needed anymore then

File: README.md [hunk 2]
```diff
@@ -12,4 +12,4 @@ Check out these milestones for the list of RFCs that are planned to be or are cu
...
 2. Create a new, appropriately named, markdown document in the `RFCs` directory, based on [Template.md](Template.md).
```
Comment (Yardanico) [inline README.md:13]: This too

File: README.md [hunk 3]
```diff
@@ -12,4 +12,4 @@ Check out these milestones for the list of RFCs that are planned to be or are cu
...
 3. Write your RFC, following the guidelines in the template.
```
Comment (Yardanico) [inline README.md:14]: I think only 3 and 4 should be there now

**Comment by hlaaftana** [comment]
Could you please update based on the comments? People are still falling for this

**Comment by xflywind** [comment]
please merge this

**Pull request closed without merging**